### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by james-lim-xq**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.
There were some typos breaking the `submit_false_negative_report_command` and `submit_false_positive_report_command`

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by mmhw** [comment]
Hi @james-lim-xq ,
Thank you for your contribution!
Good work :)

Please fill out the [Contribution Form Filled](https://forms.gle/XDfxU4E61ZwEESSMA).


Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by james-lim-xq** [comment]
> Hi @james-lim-xq , Thank you for your contribution! Good work :)
> 
> Please fill out the [Contribution Form Filled](https://forms.gle/XDfxU4E61ZwEESSMA).
> 
> Please feel free to reach out to me with any questions - I'm available here or on slack :) Thanks again

Hi @mmhw, I was checking our partner ID. I've filled out the form. Let me know if you need me to do anything else before merging!

File: Packs/AbnormalSecurity/ReleaseNotes/2_2_8.md
Comment (mmhw) [inline Packs/AbnormalSecurity/ReleaseNotes/2_2_8.md:7]:
```suggestion
- Fixed an issue in the **abnormal-security-submit_false_negative_report** command where the *recipient_email* argument did not work as expected.
```

File: Packs/AbnormalSecurity/ReleaseNotes/2_2_8.md
Comment (mmhw) [inline Packs/AbnormalSecurity/ReleaseNotes/2_2_8.md:6]:
```suggestion
- Fixed an issue in the **abnormal-security-submit_false_positive_report** command where the *portal_link* argument did not work as expected.
```

**Review by mmhw — COMMENTED** [review]
Hi @james-lim-xq  Note that the customers are not so interested in what the problem was in the code, but how it affected them in XSOAR.

Other than that everything is fine, thanks again!

**Comment by james-lim-xq** [comment]
Hi @mmhw I applied the changes requested! Please help to approve if all is good. Thank you!

**Review by mmhw — APPROVED** [review]

**Pull request merged**
